### PR TITLE
add a new modeling rule to shodan pack

### PR DESCRIPTION
**Pull request by AviShakuri**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CRTX-137200

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Shodan/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Shodan Banner
+ 
+<~XSIAM>
+ 
+This pack includes Cortex XSIAM content.
+ 
+## Configuration on Server Side
+Setup
```
Comment (ShirleyDenkberg) [inline Packs/Shodan/README.md:8]:
```suggestion

```

File: Packs/Shodan/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+To enable the Shodan integration you need to have an API key, 
+which you can get for free by creating a Shodan account https://account.shodan.io/register 
+Once you have an API key you insert it into the API Key field and click the Test button.
+
+## Configuration on Cortex XSIAM 
+You will need to use the information described: [here](https://xsoar.pan.dev/docs/reference/integrations/shodan-v2).
```
Comment (ShirleyDenkberg) [inline Packs/Shodan/README.md:14]:
```suggestion
You will need to use the information described [here](https://xsoar.pan.dev/docs/reference/integrations/shodan-v2).
```

File: Packs/Shodan/README.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ 
+<~XSIAM>
+ 
+This pack includes Cortex XSIAM content.
+ 
+## Configuration on Server Side
+Setup
+To enable the Shodan integration you need to have an API key, 
+which you can get for free by creating a Shodan account https://account.shodan.io/register 
+Once you have an API key you insert it into the API Key field and click the Test button.
```
Comment (ShirleyDenkberg) [inline Packs/Shodan/README.md:11]: Is this done (API Key field, Test button) on the XSIAM side?

**Comment by ShirleyDenkberg** [comment]
@AviShakuri Doc review completed.

**Review by eepstain — APPROVED** [review]

**Pull request merged**